common/src/main/java/org/apache/celeborn/common/network/client/TransportClientFactory.java

```diff
@@ -176,6 +176,9 @@ public TransportClient createClient(
         }
       }
       clientPool.clients[clientIndex] = internalCreateClient(resolvedAddress, decoder);
+      logger.debug(
+          "local client channel address {}",
```

Comment:
Please enrich the message, it's confusing what happened here on reading message.

Reply:
removed, and added details local channel address when the client return.